lib/storage.js

```diff
@@ -441,6 +441,42 @@ exports.set = function(key, json, options, callback, retries) {
   });
 };
 
+/**
+ * @summary Write user data sync
+ * @function
+ * @public
+ *
+ * @param {String} key - key
+ * @param {Object} json - json object
+ * @param {Function} callback - callback (error)
+ *
+ * @example
+ * const storage = require('electron-json-storage');
+ *
+ * storage.setSync('foobar', { foo: 'bar' }, function(error) {
+ *   if (error) throw error;
+ * });
+ */
+exports.setSync = function(key, json, options, callback) {
+    const fileName = utils.getFileName(key, {
+      dataPath: options.dataPath
+    });
+    const data = JSON.stringify(json);
+
+    if (!data) {
+      return callback(new Error('Invalid JSON data'));
+    }
+
+    try {
+      mkdirp.sync(path.dirname(fileName));
+      fs.writeFileSync(fileName, data);
+      return callback();
```

Comment:
No need to have `callback` and call it if the call is synchronous. We can just `return` and `throw` instead